### ISSUE TITLE
Fatal error prevents upgrade

### ISSUE THREAD

**Issue by spaceone** [PapooSoftware/PapooCMS#17]
I tried to upgrade from Version 22.08 Rev. a97990a to 23.04 by copying the `update/` directory with all the sql files into `/` (filemode user:www-data 0666) and a click to `/interna/update.php?menuid=65&start_update=2` but the following exception happens:

```
Fatal error  Uncaught TypeError: implode(): Argument #1 ($array) must be of type array, string given in /var/www/html/lib/classes/update_class.php:255
Stack trace:
#0 /var/www/html/lib/classes/update_class.php(255): implode()
#1 /var/www/html/lib/classes/update_class.php(140): update_class-&gt;new_update()
#2 /var/www/html/lib/classes/intern_stamm_class.php(184): update_class-&gt;make_update()
#3 /var/www/html/lib/classes/intern_stamm_class.php(170): intern_stamm_class-&gt;make_update()
#4 /var/www/html/interna/all_inc.php(118): intern_stamm_class-&gt;make_inhalt()
#5 /var/www/html/interna/update.php(17): require_once('...')
#6 {main}
  thrown in <b>/var/www/html/lib/classes/update_class.php</b> on line <b>255
```

**Comment by cz-papoo** [comment]
The `TypeError` has been fixed with commit 92f6d82. It was old code that is no longer supported as of PHP 8.0.

Try the latest patch and check if the problem persists.

The update routine hasn't been tested in a long time and is not guaranteed to work. If you still experience problems while upgrading, try using the latest database structure (setup/papoo.sql) and import the data of your current installation. Remember to have a backup ready, or better yet, run the upgrade on a local copy.

Note that the Papoo CMS is old software and doesn't receive regular updates anymore. If you receive any warnings using the latest commit of the main branch, try disabling PHP warnings and notices in the server configuration.

**Comment by spaceone** [comment]
Thank you!
